tracer/test/Datadog.Trace.Security.Unit.Tests/EncoderUnitTests.cs

```diff
@@ -12,6 +12,7 @@
 
 namespace Datadog.Trace.Security.Unit.Tests;
 
+[Collection("SecuritySequentialTests")]
```

Comment:
Instead of updating all these tests, I think you could add the `[Collection]` attribute to the `WafLibraryRequiredTest` base class - that way they all inherit it, and you don't have to remember it for new implementations

Reply:
You are right. Nice trick